client/client.go

```diff
@@ -478,14 +478,20 @@ func (r *NotaryRepository) GetTargetByName(name string, roles ...string) (*Targe
 
 }
 
+// TargetSignedStruct is a struct that contains information about the Target and the list of signatures
+type TargetSignedStruct struct {
```

Comment:
You're right, I misunderstood @endophage, working on a fix right now. It should be up soon :)